descriptor.md

```diff
@@ -151,6 +151,20 @@ Implementations MAY implement SHA-512 digest verification for use in descriptors
 When the _algorithm identifier_ is `sha512`, the _encoded_ portion MUST match `/[a-f0-9]{128}/`.
 Note that `[A-F]` MUST NOT be used here.
 
+## Embedded Content
+
+In many contexts, such as when downloading content over a network, resolving a descriptor to its content has a measurable fixed "roundtrip" latency cost.
+For large blobs, the fixed cost is usually inconsequental, as the majority of time will be spent actually fetching the content.
+For very small blobs, the fixed cost can be quite significant.
+
+Implementations MAY choose to embed small pieces of content directly within a descriptor to avoid roundtrips.
+
+Implementations SHOULD NOT populate the `data` field in situations where doing so would unexpectedly modify content identifiers.
```

Comment:
This seems like a MUST

Reply:
Or at least there is nothing "unexpectedly" about it. It is completely"expected" that the digest changes if this kind of modification occurs

Reply:
Changed to MUST, also added a contrasting example to demonstrate that clients _do_ have this flexibility.